src/lib/character.js

```diff
@@ -0,0 +1,51 @@
+export function createCharacter() {
+  let url;
+  const filterDropdown = document.querySelector(".header__filter");
+  const type = filterDropdown.value;
+
+  const search = document.querySelector(".header__search-form");
+  const searchInput = search.value;
+
+  if (type === "alive") {
+    url = `https://rickandmortyapi.com/api/character/?name=${searchInput}&status=alive`;
+  } else if (type === "dead") {
+    url = `https://rickandmortyapi.com/api/character/?name=${searchInput}&status=dead`;
+  } else if (type === "unknown") {
+    url = `https://rickandmortyapi.com/api/character/?name=${searchInput}&status=unknown`;
+  } else if (type === "all") {
+    url = `https://rickandmortyapi.com/api/character/?name=${searchInput}`;
+  }
```

Comment:
You could put this in a function like `getUrlFrom(type)` and put the function at the bottom of the file and call it here with passing the `type` to get just the right url back.